src/arch/riscv/linux/fs_workload.cc

```diff
@@ -196,5 +260,11 @@ BootloaderKernelWorkload::unserialize(CheckpointIn &checkpoint)
     kernelSymbolTable.unserialize("kernel_symbol_table", checkpoint);
 }
 
+void
+Gem5Exit::process(ThreadContext *tc)
+{
+    fatal(descr());
```

Comment:
I think this should be a `panic` since it's not an incorrect configuration. 

Really, I'm not sure if it should be a `panic` *or* a `fatal`. What is this purpose of this event?

Reply:
I think the best way to describe this event is to exit gem5 when there's a kernel panic.

I leaned towards using `fatal` because `panic` implies there's something with gem5. A kernel panic could also be a product of a wrong configuration, e.g., using a wrong boot parameter.

Another reason is that `panic` dumps the backtrace while `fatal` does not. The backtrace might be misleading as the exit message is pushed further upward.

Anyway, if `fatal` means there __must__ be something wrong with the configuration then I'll switch to use `panic`.

Reply:
If the goal is to exit the gem5 simulation loop, then you should call `exitSimLoop` (this isn't the exact name of the function, but it's something like that). I agree that neither `panic` nor `fatal` is right in this case.

Reply:
+1 it's `exitSimLoop` from `sim/sim_exit.hh`.

Reply:
Thanks for the suggestions. I updated the exit event accordingly.

I want to note that, for ArmFsWorkloads, guest's kernel panic will cause a gem5 panic, so we have different kernel panic exit behaviors for workloads of different ISAs. This might be confusing to users.